corehq/apps/export/models/new.py

```diff
@@ -1238,14 +1268,16 @@ class SplitUserDefinedExportColumn(ExportColumn):
     )
     user_defined_options = ListProperty()
 
-    def get_value(self, doc, base_path, transform_dates=False, **kwargs):
+    def get_value(self, domain, doc_id, doc, base_path, transform_dates=False, **kwargs):
```

Comment:
yeah, we get sub documents from the actual document: https://github.com/dimagi/commcare-hq/blob/985908e8513427697f52745b8e2e3af68bbfa5f4/corehq/apps/export/models/new.py#L312. so things like repeat groups will just return the inner context of a repeat group rather than the whole form.  better name for doc might help